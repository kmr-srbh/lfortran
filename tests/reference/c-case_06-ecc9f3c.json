{
    "basename": "c-case_06-ecc9f3c",
    "cmd": "lfortran --no-color --show-c {infile}",
    "infile": "tests/../integration_tests/case_06.f90",
    "infile_hash": "3987983e4c8d162530bceffbbce0bd7d038b1f794655c222c47ad714",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "c-case_06-ecc9f3c.stderr",
    "stderr_hash": "f7a8a727133194afebc82e107b131805af9618c10eb231882f559745",
    "returncode": 2
}