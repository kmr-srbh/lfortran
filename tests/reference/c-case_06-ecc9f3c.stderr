semantic error: The RHS character len must be known at compile time
  --> tests/../integration_tests/case_06.f90:13:5
   |
13 |     character, parameter :: grade_fixed = 'B'
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
